### PR TITLE
added FEDERATION.md / FEP-67ff

### PR DESCRIPTION
**Pull request by tobiasd**

### CODE REVIEW

File: FEDERATION.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Federation
+
+## Supported federation protocols and standards
+
+- [ActivityPub](https://www.w3.org/TR/activitypub/) (Server-to-Server)
```
Comment (annando) [inline FEDERATION.md]: We also support server to client.

File: FEDERATION.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+- [FEP-61cf: The OpenWebAuth Protocol](https://codeberg.org/fediverse/fep/src/branch/main/fep/61cf/fep-61cf.md) - basics to log in to Hubzilla
+- [FEP-67ff: FEDERATION.md](https://codeberg.org/fediverse/fep/src/branch/main/fep/67ff/fep-67ff.md)
+
+## ActivityPub
+
+<!-- Describe activities and extensions. -->
+
+## Diaspora protocol
+
+<!-- Describe activities and extensions. -->
```
Comment (annando) [inline FEDERATION.md]: You could add something like:
> Friendica supports most entities of the Diaspora protocol except polls.

File: FEDERATION.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+- [Diaspora* Protocol](https://diaspora.github.io/diaspora_federation/)
+- [DFRN](https://git.friendi.ca/friendica/friendica/src/branch/develop/spec)
+
+## Supported FEPs
+
+- [FEP-f1d5: NodeInfo in Fediverse Software](https://codeberg.org/fediverse/fep/src/branch/main/fep/f1d5/fep-f1d5.md)
+- [FEP-1b12: Group federation](https://codeberg.org/fediverse/fep/src/branch/main/fep/1b12/fep-1b12.md) - basics for federation with lemmy, kbin
+- [FEP-2677: Identifying the Application Actor](https://codeberg.org/fediverse/fep/src/branch/main/fep/2677/fep-2677.md)
+- [FEP-e232: Object Links](https://codeberg.org/fediverse/fep/src/branch/main/fep/e232/fep-e232.md)
+- [FEP-f1d5: NodeInfo in Fediverse Software](https://codeberg.org/fediverse/fep/src/branch/main/fep/f1d5/fep-f1d5.md)
```
Comment (annando) [inline FEDERATION.md]: Nodeinfo is used twice here.

File: FEDERATION.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+- [FEP-61cf: The OpenWebAuth Protocol](https://codeberg.org/fediverse/fep/src/branch/main/fep/61cf/fep-61cf.md) - basics to log in to Hubzilla
+- [FEP-67ff: FEDERATION.md](https://codeberg.org/fediverse/fep/src/branch/main/fep/67ff/fep-67ff.md)
+
+## ActivityPub
+
+<!-- Describe activities and extensions. -->
```
Comment (annando) [inline FEDERATION.md]: Please add:

> We send a `follow` activity for the id of a received root post. This is meant as a request to be included in the collection of receivers for this specific post.

**Review by annando — CHANGES_REQUESTED** [review]

File: FEDERATION.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+## Diaspora protocol
+
+Friendica supports most entities of the Diaspora protocol except polls.
+
+## Additional documentation
+
+- Documentation is available at every Friendica node at `/help` and in the project repository [friendica/doc](https://git.friendi.ca/friendica/friendica/src/branch/develop/doc) (links work on the nodes documentation).
+
+<!-- Add links to documentation pages. -->
```
Comment (annando) [inline FEDERATION.md]: Can you remove this line?

**Comment by tobiasd** [comment]
I've created a PR for including Friendica in the implementations list of the [FEP](https://codeberg.org/fediverse/fep/pulls/385)

**Pull request merged**